app/client/components/payment/update/dd/directDebitInputForm.tsx

```diff
@@ -195,11 +195,22 @@ export const DirectDebitInputForm = (props: DirectDebitUpdateFormProps) => {
           />
         </FieldWrapper>{" "}
       </div>
+
       <Checkbox
-        onChange={(newValue) => setSoleAccountHolderConfirmed(newValue)}
+        onChange={(e) => setSoleAccountHolderConfirmed(e.target.checked)}
         checked={soleAccountHolderConfirmed}
         label="I confirm that I am the account holder and I am solely able to authorise debit from the account"
         required
+        value="I confirm that I am the account holder and I am solely able to authorise debit from the account"
+        cssOverrides={css`
+          div {
+            padding: 10px 0;
```

Comment:
Could this padding be applied elsewhere? It feels better not to touch the internal structure of Source components as it's liable to change with new versions. (I've pulled down the branch and can't see this being applied to anything within the checkbox, unless I'm looking in the wrong place?)

Reply:
<img width="858" alt="Screenshot 2022-01-05 at 13 32 07" src="https://user-images.githubusercontent.com/91546670/148225963-0f637035-ef54-4818-873d-72f910b5d3b9.png">

So the source component has a min-height of 44px, this means on desktop, when the checkbox label is clamped to one line, it acts like it has a padding of 10px vertically. 

<img width="346" alt="Screenshot 2022-01-05 at 13 32 21" src="https://user-images.githubusercontent.com/91546670/148227419-47464f6b-6627-478f-82f3-42a4b222ea46.png">

On mobile the component is taller because of the line breaks so there's no padding. That being said, padding is a bit odd so I'm going to wrap the source component in a div and add margin instead just for mobile. Also, note in the screenshot that if you go really small, the tick is oddly positioned. Does this happen on yours?

Reply:
yeah, somethings not right here actually. Basically the source component has a min-height of 44px so it's being a bit tricky to work with. The vertical padding/margin needs to be 24px.

<img width="858" alt="Screenshot 2022-01-05 at 13 32 07" src="https://user-images.githubusercontent.com/91546670/148228673-191eb2b8-f297-4b00-8b49-fec2953e5e62.png">

Also, when the checkbox label is clamped to 3 lines on very small screens, the tick is out of place. happening both with and without the 10px padding I applied, does it happen on yours?

<img width="346" alt="Screenshot 2022-01-05 at 13 32 21" src="https://user-images.githubusercontent.com/91546670/148228890-67392a00-1328-44e6-a15c-f2eebfab5ed6.png">

Reply:
So I need to sort the margin out, and then also make the font smaller on mobile to avoid the clamping issue?

Reply:
Yeah, I'm seeing the same thing with the tick when the label wraps on to multiple lines. I'm not sure there's much you can do about that though as it looks like a bug in the Source component itself. Was wondering if it's fixed in a newer version, but still seeing it in v3 and v4.

Reply:
It's because the checkbox and label are vertically centred, but the tick is absolutely positioned with `top: 15px`. Overriding that with `top: calc(50% - 8px);` does the job.

Reply:
I fixed the vertical margin if you want to pull the branch again, overriding the source component styles don't seem to be working though, could you try please?

Reply:
Managed to figure this one out:

```jsx
cssOverrides={css`
  & + span {
    top: calc(50% - 8px);
  }
`}
```

`cssOverrides` targets the `input` element directly and the `span` containing the tick is at the same level in the DOM.